src/Platforms/DB2Platform.php

```diff
@@ -974,8 +976,15 @@ public function prefersIdentityColumns()
         return true;
     }
 
+    public function createSelectSQLBuilder(): SelectSQLBuilder
+    {
+        return new DefaultSelectSQLBuilder($this, 'WITH RR USE AND KEEP UPDATE LOCKS', null);
+    }
+
     /**
      * {@inheritDoc}
+     *
+     * @deprecated This API is not portable.
```

Comment:
you should also add the `Deprecation::trigger()` call in this method (the call in the parent method won't be triggered as the method is overridden)